lib/logger/README.md

```diff
@@ -0,0 +1,57 @@
+# logger
+
+The `logger` module provides an object that wraps various console logging methods safely. The default level is INFO, but it is recommended that you set this according to your environment.
+
+## Usage
+
+```javascript
+var Logger = require('bv-ui-core/lib/logger');
+// <method> is one of the 5 types, ex: log
+Logger.<method>('Hello, world!') // console.log('Hello, world!')
+```
+
+### #setLogLevel
+
+Set the log level to one of the following to prevent logging under the specified level.
+```
+Logger.DEBUG
+Logger.LOG
+Logger.INFO
+Logger.WARN
+Logger.ERROR
+Logger.OFF
+```
+
+```javascript
+var Logger = require('bv-ui-core/lib/logger');
+
+// Set's the default log level to one of the valid levels
+// DEBUG, LOG, INFO, WARN, ERROR, OFF
+Logger.setLogLevel(Logger.DEBUG)
+
+Logger.log('Hello, World!') // Prints Hello, World!
+
+Logger.setLogLevel(Logger.INFO)
+
+Logger.log('Hello, World!') // Prints nothing because of logLevel
+```
+
+### group/groupEnd
+https://developer.mozilla.org/en-US/docs/Web/API/console/group
```

Comment:
This is a little nitpicky but just to let you know, it wasn't immediately clear to me that group was a method I could call on Logger. Then I realized it was implied that each sub-header here is a method. Also, the first one starts with a '#', the others don't.